README.md

```diff
@@ -16,6 +16,21 @@ The code in the example above comes from the excellent [manim tutorial](https://
 ```sh
 pip3 install jupyter-manim
 ```
+### Installation in Anaconda:
+Starting point: there are two anaconda envs:
```

Comment:
Is this going to be the case for everyone, or is it just how you installed `manim` and `jupyter`? While I do see how an anaconda instructions might be helpful for someone, I am not sure that everyone will have the same starting conditions; no to mention, that many people use miniconda without navigator etc.

Also, I do believe that most conda users should understand how the isolation of environments work and why they may need to install the required packages in a single environment - this is the entire point of using conda. I see how a guide could help the novice users, but I am afraid that the task of creating a comprehensive instruction set (i.e. one which would work for everyone and be more helpful than confusing) is beyond our ability here - this would be better solved by linking to a maintained guide on the anaconda website.

Reply:
Alright,
this is just how I got this to work.
And youre right, probably people can figure that out by themselves.